trueos-handbook/install.rst

```diff
@@ -9,6 +9,290 @@ graphical desktop directly onto a hard drive or into a virtual machine
+   FreeBSD. This means users see less experimental work and generally
+   have a smoother experience. However, users on STABLE also typically
+   wait longer for bugfixes and patches to be available. While some
+   |trueos| development may be backported to STABLE early, FreeBSD
+   patches and port synchronization is done on a six-month schedule.
+
+2. UNSTABLE is the full leading edge of TrueOS and FreeBSD development.
+   Patches are very frequent, but can incorporate experimental work from
+   both the |trueos| and FreeBSD projects. UNSTABLE is recommended for
+   users who need the absolute latest work from FreeBSD or |trueos| and
+   are willing to tolerate breakage or less system stability. It is also
+   recommended for users who want to test and contribute patches to
+   FreeBSD or |trueos|.
+
+Periodically, the |sysadm| :sysclbk:`Update Manager <update-manager>`
+provides patches to update the operating system. By default, users who
+install STABLE receive updates from the STABLE track, and UNSTABLE users
+are on the UNSTABLE track. It is possible to switch update tracks
+post-installation. See the :ref:`Updating TrueOS` section for
+instructions on switching update repositories.
+
+Installation files can be downloaded from the
+`TrueOS® website <https://www.trueos.org/downloads/>`_ or the
+`PC-BSD® CDN <http://iso.cdn.pcbsd.org/>`_.
+
+Several types of files are available for download. Any file with the
+naming convention *TrueOS-<year>-<month>-<day>-x64-[DVD/USB].[iso/img]*
+are STABLE or UNSTABLE installation files, depending on the download
+directory:
+
+* `STABLE <http://download.trueos.org/master/amd64/>`_: Files in the
+  :file:`master/amd64` directory are the STABLE version of |trueos|.
+
+* `UNSTABLE <http://download.trueos.org/unstable/amd64/>`_: Files in the
+  :file:`unstable/amd64` directory are the UNSTABLE version of |trueos|.
+
+* **"latest" files**: These entries are symlinks to their related
+  install files and are generally ignored.
+
+Install files following the above naming conventions can also end with a
+variety of extensions:
+
+* **.iso**: If the file has an *.iso* extension, it should be burned to
+  a DVD media.
+* **.img**: If it has a *img* extension, it should be burned to a USB
+  stick.
+* **.md5, .sha256, and .sig**: Depending upon the current operating
+  system and its tools, use the value in any of these files to
+  determine the integrity of the download, as described in
+  :ref:`Data Integrity Check`.
+* **.torrent**: If a torrent is available, a file with the same name
+  and a *.torrent* extension will be visible.
+
+To install a graphical desktop, download the file ending in *DVD.iso* or
+*USB.img*. Then, depending on the file type, either burn it to a DVD
+media or write it to a removable USB device.
+
+If installing a command-line only server is preferred, download and
+begin installing |trueos| in the same manner as the desktop, but choose
+the :guilabel:`Server` option in the installer.
+
```

Comment:
Server installation text may need to be reworked; there may be a separate installation file for TrueOS server now.